### DISCUSSION TITLE
Running two or more python scripts in parallel. Does prefect provide BashOperator like in Airflow?

### DISCUSSION THREAD

**Discussion by Sinha-Ujjawal** [PrefectHQ/prefect#3365] [Q&A] [answered]
So, I have this requirement of running 15 python scripts (that reads data from AWS S3 and process it and stores it in Postgres), these scripts are already developed by other people. I don't have time to refactor them and create individual prefect tasks. Does prefect provide BashOperator like in Airflow?

e.g.- something like this
``` python
BashOperator(command="python task1.py")
```

**Comment by joshmeek** [comment, marked as answer]
Hi @Sinha-Ujjawal to perform something like this in Prefect look into using the ShellTask https://docs.prefect.io/api/latest/tasks/shell.html#shelltask

**Reply by Sinha-Ujjawal** [reply]
Thx for the reply 👍
